internal/testdata/circular_fk_models/models.go

```diff
@@ -0,0 +1,13 @@
+package circular_fk_models
```

Comment:
```suggestion
package circularfks
```
don't use _ in package names.